src/templates/finops-hub/modules/dataFactory.bicep

```
@@ -163,6 +168,11 @@ resource keyVault 'Microsoft.KeyVault/vaults@2023-02-01' existing = if (!empty(r
   name: keyVaultName
 }
 
+// Get ADX cluster instance
+resource ADXcluster 'Microsoft.Kusto/clusters@2023-08-15' existing = {
```

nit: Call this `dataExplorer` or `dataExplorerCluster` for consistency with other places.
```suggestion
resource dataExplorer 'Microsoft.Kusto/clusters@2023-08-15' existing = {
```

---

ok, checked an know how to adjust it